src/files/FilesPage.js

```diff
@@ -34,12 +41,27 @@ class FilesPage extends React.Component {
   }
 
   onRename = ([path]) => {
-    const oldName = path.split('/').pop()
-    const newName = window.prompt('Insert the new name:')
+    this.setState({
+      rename: {
+        show: true,
```

Comment:
Don't want to nitpick, but I would rename this to something like `isOpen`, that way it makes it clearer that it's a Boolean.

But I leave it up to you 🙂

Reply:
Yeah, that's a great idea. Thanks! 😄